README.md

```diff
@@ -1,239 +1,233 @@
+Add up the items of a bill, then add sales tax:
 
 ```js
 function calculateBill(items, tax) {
-  let total = 0m;
+  let total = new Decimal("0");
   for (let {price, count} of items) {
-    total += price * BigDecimal(count);
+    total = total.add(price.times(new Decimal(count)));
   }
-  return BigDecimal.round(total * (1m + tax),
-                          {maximumFractionDigits: 2, round: "up"});
+  return total.multiply(tax.add(1));
 }
 
-let items = [{price: 1.25m, count: 5}, {price: 5m, count: 1}];
-let tax = .0735m;
-console.log(calculateBill(items, tax));
+let items = [{price: "1.25", count: 5}, {price: "5.00", count: 1}];
+let tax = "0.0735";
+console.log(calculateBill(items, tax).toFixed(2));
 ```
 
 #### Why use JavaScript for this case?
 
-Historically, JavaScript may not have been considered a language where these representations and manipulations are necessary. In some application architectures, JS only deals with a string representing a human-readable decimal quantity, and never do calculations or conversions. However, several trends push towards JS's deeper involvement in with decimal quantities:
-- **More complicated frontend architectures**: Rounding, localization or other presentational aspects may be performed on the frontend for better interactive performance
-- **Serverless**: Many Serverless systems use JavaScript as a programming language in order to better leverage the knowledge of frontend engineers
-- **Server-side programming in JavaScript**: Systems like Node.js and Deno have grown in popularity to do more traditional server-side programming in JavaScript
+Historically, JavaScript may not have been considered a language where exact decimal numbers are even representable, to say nothing of doing (exact) calculations. In some application architectures, JS only deals with a string representing a human-readable decimal quantity, and never do calculations or conversions. However, several trends push towards JS’s deeper involvement in with decimal quantities:
 
-In all of these environments, the lack of Decimal support means that various workarounds have to be used:
-- An external library could be used instead (introducing issues about choosing the library, coordinating on its use)
-- Calculations could be in terms of "cents" (fallible as explained above)
-- In some cases, developers end up using Number instead, believing it to be mostly safe, but in practice causing bugs
++ **More complicated frontend architectures**: Rounding, localization or other presentational aspects may be performed on the frontend for better interactive performance.
++ **Serverless**: Many Serverless systems use JavaScript as a programming language in order to better leverage the knowledge of frontend engineers.
++ **Server-side programming in JavaScript**: Systems like Node.js and Deno have grown in popularity to do more traditional server-side programming in JavaScript.
```

Comment:
These two points do not mention why serverless or server-side would need decimal, just that they exist.

Reply:
Right. I think the point to make here is that JS is on the rise, appearing in places where it traditionally has not been used. This increases, theoretically, the need for handling decimals. (We're trying to argue that decimals are needed because the days are long gone when JS was "just in the browser, doing trivial stuff".)